modules/gschema-overrides/README.md

```diff
@@ -0,0 +1,38 @@
+# `gschema-overrides` module for startingpoint
+
+`Gschema-overrides` startingpoint module is used for including your system-setting overrides.
```

Comment:
- not a startingpoint module, that name will be retired soon
- sentence structure should be same as other modules
- what environmnets use gschemas?

Something like this would be better:

> The `gschema-overrides` can be used for including system settings overrides on `<supported environment 1>` and `<supported environment 2>` .

Reply:
1. Will rename it to BlueBuild module
2. Hmm, I tried to follow the same structure by including how the module works then including how it should be used with "Example Configuration". Is there something I'm missing?
3. Gnome-based DEs only. Will correct this.

Reply:
Updated.

Only 2nd point is not resolved.

Reply:
What you've got going on right now is good. With sentence structure I meant the "The `blahblah` module offers/can be used/etc" part.

Reply:
I'm not sure about using this style convention everywhere as it can sound worse & more confusing to the user depending on the sentence & context.

At least it looks like that to me when I started to reword current sentences with your suggestions.

Reply:
> I'm not sure about using this style convention everywhere as it can sound worse & more confusing to the user depending on the sentence & context.

Feel free to make an issue about that presenting your views and possible other conventions that could be adopted.